style.css

```diff
@@ -0,0 +1,37 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+    color: rgb(14, 166, 241);
+}
+
+body{
+    display: flex;
+    justify-content: center;
+    align-self: center;
```

Comment:
```suggestion
    display: grid;
    Place-items: center;
```

Reply:
but on applying grid instead of flex - autofill property not working...